bin/bst-speak.ts

```diff
@@ -30,62 +27,35 @@ program
         }
 
         // Just by casting program to options, we can get all the options which are set on it
-        let options: any = program;
-        let url = options.url;
-        let intentSchemaPath = options.intents;
-        let samplesPath = options.samples;
-        let applicationID = options.appId;
+        const options: any = program;
+        const token = options.token;
 
-        if (options.url === undefined) {
-            let proxyProcess = Global.running();
-            if (proxyProcess === null) {
-                console.error("No URL specified and no proxy is currently running");
+        let silentEchoResponse;
+        try {
+            silentEchoResponse = await SilentEchoClient.speak(utterance, token);
+        } catch (error) {
+            if (error.message === "Token Required") {
+                console.log("You need a token for this option to work, get it here:");
                 console.log();
-                console.log("URL (--url) must be specified if no proxy is currently running");
+                console.log("\thttps://apps.bespoken.io/dashboard/skills/" + Global.config().sourceID() + "/validation");
```

Comment:
Same as above

Reply:
you mean we shouldn't autocomplete the skill id?